src/character.cpp

```diff
@@ -10943,22 +10942,15 @@ void Character::process_effects()
                     if( !has_effect( effect_cramped_space ) ) {
```

Comment:
It's still there

Reply:
Sorry, I misread your review and pulled the wrong thing. Now it's fixed.